sew.lic

```diff
@@ -360,7 +382,8 @@ class Sew
               'You are already knitting',
               'Roundtime',
               'That tool does not seem suitable for that task.',
-              'You need a larger amount of material')
+              'You need a larger amount of material',
+              'is not here!')
```

Comment:
Food for thought: This could be added to bput itself to make it auto-retry, much like it does with playing an instrument.

Reply:
Maybe that will just result in spam though?

Reply:
I've been thinking about it, but it's just such a weird issue.  This only seems to happen inside caravans, and only with PUSH, but the only time I'd know how to fix it is if you're using MY.  It's not a temporary thing, either, it always fails if one of these My- people is online.

I should check to see if PUSHing any other word fragments give this error, or if it's just 'my'.  I bet there are some other things in people's persons that you'd need to PUSH from time to time, but if that problem crops up for other words then I won't know what to replace it with.

Reply:
Oh, I see it now.  PUSH Person is syntax to kick someone out of the caravan.  Of course it looks at who's online.  So Push anything would probably give the same error.

Makes sense now.  Just a parser priority problem